ews-java-api.iml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="false">
+  <component name="FacetManager">
+    <facet type="Osmorc" name="OSGi">
+      <configuration manifestGenerationMode="OsmorcControlled" manifestLocation="" jarfileLocation="ews-java-api-1.3-SNAPSHOT.jar" outputPathType="CompilerOutputPath" bndFileLocation="" bundlorFileLocation="" bundleActivator="" bundleSymbolicName="com.microsoft.office.ews-java-api" bundleVersion="1.3.0.SNAPSHOT" ignoreFilePattern="" useProjectDefaultManifestFileLocation="true" alwaysRebuildBundleJAR="false" doNotSynchronizeWithMaven="false">
+        <additionalProperties>
+          <property key="Bundle-Description" value="Exchange Web Services (EWS) Java API" />
+          <property key="Bundle-Vendor" value="Microsoft" />
+          <property key="Bundle-DocURL" value="http://www.microsoft.com/" />
+        </additionalProperties>
+        <additionalJARContents />
+      </configuration>
+    </facet>
+  </component>
```

Comment:
ok. But as I see there is some information added like the jar-filename. This will result in once the version-number will be changed this file needs to be changed too. IntelliJ will also just update the file if the project is launched with Osmorc plugin enabled etc.